examples/analog_hamiltonian_simulation/04_Running_Analog_Hamiltonian_Simulation_with_local_simulator.ipynb

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
<p>I would argue that we don't need cost tracking, since the notebook does not use the Braket service, hence it does not incur cost beyond local compute. (In fact, if someone is running this cost tracker on a notebook instance, they may get the wrong impression that the instance is given at no charge, which is not true.)</p><p> </p><p>We also don't have the cost tracker for notebook 00 for the same reason.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>